scripts/action-recognition/README.md

```diff
@@ -0,0 +1,65 @@
+# Action Recognition[1]
+[GluonCV Model Zoo](https://gluon-cv.mxnet.io/model_zoo/action_recognition.html)
+
+## Inference/Calibration Tutorial
+
+### FP32 inference
+
+```
+
+export CPUs=`lscpu | grep 'Core(s) per socket' | awk '{print $4}'`
+export OMP_NUM_THREADS=${CPUs}
+export KMP_AFFINITY=granularity=fine,noduplicates,compact,1,0
+
+# dummy data
+python test_recognizer.py --model inceptionv3_ucf101 --use-pretrained --use-tsn --mode hybrid --input-size 299 --new-height 340 --new-width 450 --num-segments 3 --batch-size 64 --benchmark
+
+# real data
+python test_recognizer.py --model inceptionv3_ucf101 --use-pretrained --use-tsn --mode hybrid --input-size 299 --new-height 340 --new-width 450 --num-segments 3 --batch-size 64
+```
+
+### Calibration
+
+Naive calibrated model by using 5 batch data (32 images per batch). Quantized model will be saved into `./model/`.
+
+```
+# ucf101 dataset
+python test_recognizer.py --model inceptionv3_ucf101 --new-height 340 --new-width 450 --input-size 299 --num-segments 3 --use-pretrained --use-tsn --calibration
+```
+
+### INT8 Inference
+
+```
+
+export CPUs=`lscpu | grep 'Core(s) per socket' | awk '{print $4}'`
+export OMP_NUM_THREADS=${CPUs}
+export KMP_AFFINITY=granularity=fine,noduplicates,compact,1,0
+
+# dummy data
+python test_recognizer.py --model inceptionv3_ucf101 --mode hybrid --input-size 299 --new-height 340 --new-width 450 --batch-size 64 --num-segments 3 --quantized --benchmark
+
+# real data
+python test_recognizer.py --model inceptionv3_ucf101 --mode hybrid --input-size 299 --new-height 340 --new-width 450 --batch-size 64 --num-segments 3 --quantized
+
+# deploy static model
+python test_recognizer.py --model ucf --deploy --model-prefix ./model/inceptionv3_ucf101-quantized-naive --input-size 299 --new-height 340 --new-width 450 --batch-size 64 --num-segments 3 --benchmark
```

Comment:
As far as I can recall, we don't have a model named `ucf`. Can you explain why you use `--model ucf`? Thank you.

Reply:
In `deploy` mode, we only need to specify `--model-prefix` for loading saved model, at this time whatever the value of `--model` is fine. However, maybe we need give a more meaningful name here, like `inceptionv3_ucf101`.

Reply:
I see. You can delete `--model` or use `inceptionv3_ucf101 ` to be clearer. I'm fine with either way.